packages/eas-cli/src/credentials/ios/credentials.ts

```diff
@@ -45,7 +45,7 @@ export interface IosAppCredentials {
     // for adhoc provisioning profiles
     devices?: AppleDevice[];
 
-    teamId?: string;
+    teamId: string;
```

Comment:
Hmmmm, I think that some credentials in our DB can have this nulled out. AFAIR this happens when the user uploads credentials manually (doesn't let EAS handle credentials). @wkozyra95 correct me if I'm wrong.

Why do you need to make this non-optional?

Reply:
Right now it's always not null, but there still might be some old credentials that do not have that field.
It's possible that quin migrated all that data and extracted  team ID from all provisioning profiles, but it would still only guarantee that teamId exists only when provisioningProfile exists, which is optional

Reply:
I've reverted this line for now. I only made it non-null because this passed the type check. But it seems that the reason it passed was that the code sets empty string as a default: `teamId: provisioningProfile.appleTeam?.appleTeamIdentifier || ''`.